### ISSUE TITLE
Question re. swift-package-list 

### ISSUE THREAD

**Issue by gaw3** [FelixHerrmann/swift-package-list#16]
I have updated SwiftPackageList to 1.4.0.  This caused a package called `swift-argument-parser` v1.0.3 to get added to the SPM.  Is this OK?

Following the new instructions, I downloaded project and ran `make update` and received the following error:

```
sh uninstall.sh
sh: uninstall.sh: No such file or directory
make: *** [update] Error 127
```

Thought I would try uninstall and install:

`make uninstall` returns `rm -f "/usr/local/bin/swift-package-list"`

`make install` returns the following:

```
swift build --configuration release
Fetching https://github.com/apple/swift-argument-parser from cache
Fetched https://github.com/apple/swift-argument-parser (0.57s)
Computing version for https://github.com/apple/swift-argument-parser
Computed https://github.com/apple/swift-argument-parser at 1.0.3 (0.62s)
Creating working copy for https://github.com/apple/swift-argument-parser
Working copy of https://github.com/apple/swift-argument-parser resolved at 1.0.3
[13/13] Build complete!
cp -f ".build/release/SwiftPackageListCommand" "/usr/local/bin/swift-package-list"
cp: .build/release/SwiftPackageListCommand: No such file or directory
make: *** [install] Error 1
```

Am I doing something wrong?

**Comment by FelixHerrmann** [comment]
Hi, sorry for the inconvenience! As far as I can tell, you have the first version of the Makefile pulled which didn't work as expected, I fixed it [here](https://github.com/FelixHerrmann/swift-package-list/commit/aaf44e793a29494484ecdf488c84a1b8defbf81e). Make sure you use the latest version, that should resolve the install issue!

To your first question regarding the `swift-argument-parser`, I added support for Mint in #13 which requires the executable target to be also under the products in the `Package.swift`. I don't think there is a way to hide this, e.g. with compiler directives...
